### ISSUE TITLE
Is this project maintained?

### ISSUE THREAD

**Issue by gustawdaniel** [FriendsOfSymfony/FOSUserBundle#2996]
I have error:

```
FOS UserBundle UserManager::__construct() must be an instance of ObjectManager
```

I have seen this post:

https://stackoverflow.com/questions/64187191/fos-userbundle-usermanager-construct-must-be-an-instance-of-objectmanager

And there was written:

"""
Aside from the FOSUserBundle being no longer maintained
"""

**Comment by pscheit** [comment]
nope

**Comment by abouet** [comment]
Just had the same problem.
should be 
```php
use Doctrine\Persistence\ObjectManager;
use Doctrine\Persistence\ObjectRepository;
```
instead of 
```php
use Doctrine\Common\Persistence\ObjectManager;
use Doctrine\Common\Persistence\ObjectRepository;
```

**Comment by acantepie** [comment]
To clarify your expectations.
Never use this bundle with Symfony 5. ( https://github.com/FriendsOfSymfony/FOSUserBundle/issues/2874 ).
This bundle is not maintened anymore.

**Comment by stof** [comment]
Support for doctrine/persistence v2 has been released as part of 2.2
